understanding/20/extended-audio-description-prerecorded.html

```diff
@@ -60,10 +60,10 @@ <h2>Examples of Extended Audio Description (Prerecorded)</h2>
       <ul>
          
          <li>
-            <em>Example 1. Video of a lecture.</em> A physics professor is giving a lecture. He makes freehand sketches on the whiteboard,
-            speaking rapidly as he draws. As soon as he has finished discussing one problem, he
-            erases the drawing and makes another sketch while continuing to speak and gesture
-            with his other hand.  The video is paused between problems, and extended audio description
+            <em>Example 1. Video of a lecture.</em> A physics professor is giving a lecture. They make freehand sketches on the whiteboard,
+            speaking rapidly as they draws. As soon as they have finished discussing one problem, they
```

Comment:
actually I question this one - singular they here might be right? Someone else may need to weigh in!

Reply:
it's my understanding that even for singular use of "they", the verb is then plural

Reply:
My bad here, it should be singular (right first time). 

We are not referring to someone whose pronouns are "they", we are referring to a third person "they". So it should be "they draw", singular. 

If we mentioned a person "Alex" and then their pronouns were "they/them" it would be right to say "as they draws".

Reply:
it seems the general agreement (from multiple sources) is that the verb is then also using the plural form (e.g. "as they draw" even when referring to "Alex")

https://www.merriam-webster.com/grammar/singular-nonbinary-they-is-or-they-are#:~:text=The%20simple%20answer%20is%20that,%E2%80%9CThey%20are%20my%20friend.%E2%80%9D